app/views/format_fotos/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div id="tariff">
+<h1>Всі формати фото</h1>
+<table style="width:100%;" border="1">
```

Comment:
@romigos all styles please move to css classes

Reply:
@romigos 
where???
and app/views/format_fotos/index.html.erb => app/views/format_**photos**/index.html.erb

Reply:
fixed file deleted